### PR TITLE
[EN] corrections of week08 notes

### PR DESCRIPTION
**Pull request by marcozullich**
Correcting typos, missing math env, capitalizations...

### CODE REVIEW

File: docs/en/week08/08-3.md [hunk 1]
```diff
@@ -16,7 +16,6 @@ To summarize at a high level, a very simple form of AE is as follows:
 
 *For a detailed explaination, refer to the notes of [Week 7]({{site.baseurl}}/en/week07/07-3/).*
 
-
```
Comment (Atcold) [inline docs/en/week08/08-3.md:19]: This one was fine. We have two empty lines before any heading. Plz revert.

**Comment by marcozullich** [comment]
Ok, will do.
So can I correct it whenever I find single lines before heading?

Il giorno mar 16 giu 2020 alle ore 03:09 Alfredo Canziani <
notifications@github.com> ha scritto:

**Comment by Atcold** [comment]
Yes, please. [Here](https://atcold.github.io/pytorch-Deep-Learning/en/about/) I've written a few guidelines.

File: docs/en/week08/08-3.md (hunk 1 above)
Comment (marcozullich) [reply docs/en/week08/08-3.md]: Done

**Comment by marcozullich** [comment]
I have reverted the newlines before header (2 cases found) and after header (1 case found). It all should be OK.
Sorry for the delay, but I had a lot of work to do in the previous week

**Comment by Atcold** [comment]
Thanks, great!

**Pull request merged**